stable/xray/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # JFrog Xray Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [4.2.1] - Aug 13, 2020
+* Added support for external rabbitmq 
+* Added support for Load Definitions for rabbitmq subchart when `rabbitmq.enabled=true` . Please refer [here](https://github.com/bitnami/charts/tree/master/bitnami/rabbitmq#load-definitions)
```

Comment:
Please add the new values to the README

Reply:
Going forward we don't support readme ( configuration )